zh-cn/yodaos-source/voice-interactive/voice-interface.md

```diff
@@ -1,157 +0,0 @@
-# voice-interface flora消息定义
```

Comment:
Why had this file been removed?

Reply:
Weird, we should have removed this file on latest revision, could you check again?

Reply:
Still marked as removed?

Reply:
Sorry, I'll revert this :p